helpers/GenerateResourcesAndImage.ps1

```diff
@@ -376,4 +383,4 @@ Function GenerateResourcesAndImage {
         }
         Write-Verbose "Cleanup completed."
     }
-}
+}
```

Comment:
Please keep newline at the end of the file

Reply:
do we have ".vscode" setting for that ?